docs/index.md

```diff
@@ -3,7 +3,7 @@ author: JeremyKelley
 ms.author: JeremyKe
 ms.date: 09/18/2017
 ms.topic: landing-page
-ms.prod: onedrive
+ms.service: one-drive
```

Comment:
That's not the name of the product... it's `onedrive` (single word, no need for hyphenation)